### PR TITLE
add installation guide to README

### PR DESCRIPTION
**Pull request by onaga1958**

### CODE REVIEW

**Comment by onaga1958** [comment]
[Добавить в readme инструкцию по установке для чистого компа](https://trello.com/c/iNa739RX/4-%D0%B4%D0%BE%D0%B1%D0%B0%D0%B2%D0%B8%D1%82%D1%8C-%D0%B2-readme-%D0%B8%D0%BD%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%86%D0%B8%D1%8E-%D0%BF%D0%BE-%D1%83%D1%81%D1%82%D0%B0%D0%BD%D0%BE%D0%B2%D0%BA%D0%B5-%D0%B4%D0%BB%D1%8F-%D1%87%D0%B8%D1%81%D1%82%D0%BE%D0%B3%D0%BE-%D0%BA%D0%BE%D0%BC%D0%BF%D0%B0)

File: README.md [hunk 1]
```diff
@@ -2,3 +2,51 @@
...
+	```
+	git clone --recursive https://github.com/dmlc/xgboost
+	cd xgboost
+	git checkout 14fba01b5ac42506741e702d3fde68344a82f9f0
+	make -j
+	cd python-package; python3 setup.py install
+	cd ../../
+	rm -rf xgboost
+	```
+5. Open in jupyter-notebook ```example/model_search.ipynb``` and run all cells. If there are no errors, everything is allright.
```
Comment (anaderi) [inline README.md]: not straightforward, better to mention as  "1) start jupyter notebook" in specific folder and 2) open browser to the address X:Y, paste token there and open notebook `model_search.ipynb`

Comment (onaga1958) [reply README.md]: Locally jupyter automatically opens in browser, doesn't it? Did you mean adding instruction for running it remotely?

**Comment by anaderi** [comment]
please rebase & merge changes with Sergey's part of README

**Comment by fratnikov** [comment]
Installation guide assumes installation of XGBoos and LightGBM of particular versions.
The regular use case is that those packages are installed already with likely some different versions.
Probably a hint about versions consistency could help users to avoid hustle?

**Pull request merged**